MaxText/convert_ckpt.py

```diff
@@ -11,16 +11,16 @@
  limitations under the License.
  """
```

Comment:
Can this one be llama_or_mistral_ckpt? Because other checkpoints have other paths.

Reply:
Yes, of course. Will update it in next commit.

Reply:
and we could override any config including say tokenizer path here: https://github.com/google/maxtext/blob/main/MaxText/pyconfig.py#L176-L195

Reply:
Hi @WendyShang, do you mind explaining a little bit more when you say `give a decode.py example to signify that we are using mistral's tokenizer`? Thank you!

Reply:
Thanks Wendy! Like we discussed offline, I will keep this in mind, and see how we could improve further when adding Mixtral inference. Resolve this for now.